### PR TITLE
jsdoc comments: tweaks for consistency

### PR DESCRIPTION
**Pull request by margaree**
- remove periods at ends of small sentences
- change required fields to have "REQUIRED:" at start
- use `@type` and `@default` where applicable
- simplify component descriptions where possible

For quotation marks, I used `"` when defining a `@default` since that's what the analyzer does with defaults it finds and I wanted to keep that consistent. For defining `@type` I used `'` because when displayed in the documentation table they look quite bulky with the double quote. However, this does take away some consistency.

### CODE REVIEW

File: components/dropdown/dropdown-content-mixin.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 	static get properties() {
 		return {
 			/**
-			 * Optionally align dropdown to either `start` or `end`. If not set, the dropdown will attempt be centred.
+			 * Optionally align dropdown to either start or end. If not set, the dropdown will attempt be centred.
+			 * @type {('start'|'end')}
```
Comment (dlockhart) [inline components/dropdown/dropdown-content-mixin.js:15]: This is a weird attribute eh... the default is `undefined` which centres it? Hmm... 🤔

File: components/inputs/input-time.js [hunk 2]
```diff
@@ -112,7 +113,8 @@ class InputTime extends LitElement {
 			 */
 			maxHeight: { type: Number, attribute: 'max-height' },
 			/**
-			 * Number of minutes between times shown in dropdown. Valid values are "five", "ten", "fifteen", "twenty", "thirty", "sixty".
+			 * Number of minutes between times shown in dropdown
+			 * @type {('five'|'ten'|'fifteen'|'twenty'|'thirty'|'sixty')}
```
Comment (dlockhart) [inline components/inputs/input-time.js:117]: We don't need to call out the `@default` here because lit-analyzer gets it from the constructor, ya?

Comment (margaree) [reply components/inputs/input-time.js:117]: Yup that's correct.

File: components/dropdown/dropdown-content-mixin.js (hunk 1 above)
Comment (margaree) [reply components/dropdown/dropdown-content-mixin.js:15]: Yeah - we could add a 'centered' and do a bit of restructuring

**Pull request merged**